logging/template-logging.md

```diff
@@ -0,0 +1,70 @@
+---
+title: template for logging
+author: The Good Docs Project 
+uri-repo: https://github.com/projectname
+company: REPLACEME
+product: <!-- Set the product parameter to add the product name here. -->
+app: <!-- Insert the name of the app here. -->
+tags: <!-- Set other global keywords here like app name and product name or any other likely labels. These are comma-separated tags.-->
+---
+
+<!--
+For help with writing and structuring a log article, see the README.adoc in the template directory for basic guidelines and links.
+Check out https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/ if you get stuck with AsciiDoc syntax.
+-->
+# {pipeline} for {app}
+
+## Description
+
+<table>
```

Comment:
Prefer when possible to use `markdown` syntax over html, in this case we can write a markdown table.

Reply:
The .adoc table was too complex to be converted in Markdown. So I used HTML for keeping the style consistent. I will ask questions on principles for converting in the US meeting.